tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -212,6 +223,14 @@ function configure_crio() {
 	for shim in "${shims[@]}"; do
 		configure_crio_runtime $shim
 	done
+
+
+	if [ "${DEBUG:-"no"}" == "yes" ]; then
+		cat <<EOF | tee -a $crio_drop_in_conf_file_debug
+[crio]
```

Comment:
What about the case where a user already has set `log_level` to some value?

Reply:
You could do the same logic you did for `containerd`

Reply:
So, CRI-O has this drop-in mechanism.  Our will just override what the users have.

Reply:
Otherwise looks good to me.

Reply:
But why? :-)

We want to override it in this case anyways, and we do that by drop-in a new file.

Reply:
Holy cow, missed the `drop_in_conf_file`, there is a clear advantage if you can read.